### DISCUSSION TITLE
Microphone

### DISCUSSION THREAD

**Discussion by UsualSuspectOne** [DrCoffey/DeepSqueak#185] [Q&A]
Hey i am very new to this

I was just wondering witch microphones would one use to record Rat Calls and Convos are there any affordable Options/Suggestions 

 I already toyed around with the Sample Calls you kindly provided and was wondering how to interpret the "Mood" of Rats based on the Calls the Software detects 

Lastly Thank you for putting in the Work and making this Technology available to all of us  interested Rat Owners

**Comment by DrCoffey** [comment]
Hey @UsualSuspectOne, Higher frequency calls (~50 kHZ) are generally associated with positive affect, while lower frequency (22kHz) flat calls are generally associated with negative affect.

Dodotronic mics are about as cheap as they get, and are decent for playing around: https://www.dodotronic.com/product/ultramic-um250k/

Audio moths are a little cheaper, but they are hard to find: https://store.groupgets.com/products/audiomoth-v1-1-0

Have fun,
Kevin

**Comment by Behroz666** [comment]
Hey Dr.Coffey it seems like Audio moths option is unavailable. Could you please help me finding some other cheap ultrasonic sensors cause the Dodotronic is so expensive with the cost of shipping to my country. I have found decent options but I'm not sure those are suitable for this use. I would appreciate any help.

**Comment by LouieVirosa** [comment]
Thanks for the recommendation! Just ordered an audiomoth. I'm homeschooling my teenagers, and we'd love to visualize and classify the USVs our pair of rats are making!
